editor-extensions/vscode/package.json

```diff
@@ -69,25 +75,25 @@
       "type": "object",
       "title": "Grain Language Server configuration",
       "properties": {
-        "grain_language_server.enableLSP": {
+        "grain.enableLSP": {
           "scope": "window",
           "type": "boolean",
           "default": true,
           "description": "Enable the language server"
         },
-        "grain_language_server.enableStatementLenses": {
+        "grain.enableStatementLenses": {
```

Comment:
@marcusroberts This option doesn't actually disable lenses anymore. Do we need to supply it somewhere? Does it send a message to the LSP server?

Reply:
It was there when performance was slow, removed now.